README.md

```diff
@@ -1,172 +1,31 @@
-[![Python package](https://github.com//actions/workflows/python-package.yml/badge.svg)](https://github.com//actions/workflows/python-package.yml)
+# Smoothed Particle Hydrodynamics (SPH) via Inductiva API
```

Comment:
There is a problem with this title.
We are actually not showing Smoothed Particle Hydrodynamics via Inductiva API. We are abstracting that away. 

If we want to keep this title, then it seems reasoable to actually have a demo of the low-level calls.

Reply:
Gonna show a low-level call and a duck boyance